src/Modules/CalcPerform.lua

```diff
@@ -742,9 +755,12 @@ local function doActorLifeManaReservation(actor)
 			output[pool.."ReservedPercent"] = m_min(reserved / max * 100, 100)
 			output[pool.."Unreserved"] = max - reserved
 			output[pool.."UnreservedPercent"] = (max - reserved) / max * 100
-			if (max - reserved) / max <= data.misc.LowPoolThreshold then
+			if (max - reserved) / max <= (pool == "Life" and data.misc.LowLifePercentage / 100.0 or data.misc.LowPoolThreshold) then
```

Comment:
No need to break the generic parts of this loop.  Do something like this (maybe pull the Sum out into a local variable):
```suggestion
			if (max - reserved) / max <= modDB:Sum("BASE", nil, "Low" .. pool .. "Percentage") / 100.0 or data.misc.LowPoolThreshold) then
```